public/components/event_analytics/explorer/datasources/datasources_selection.tsx

```diff
@@ -114,7 +113,7 @@ export const DataSourceSelection = ({ tabId }: { tabId: string }) => {
   const dispatch = useDispatch();
   const routerContext = useContext(LogExplorerRouterContext);
   const explorerSearchMetadata = useSelector(selectSearchMetaData)[tabId];
-  const [activeDataSources, setActiveDataSources] = useState<DataSourceType[]>([]);
+  const [activeDataSources, setActiveDataSources] = useState<DataSource[]>([]);
```

Comment:
Could you give me more details about this change?

Reply:
This fixes type error in the code